src_c/polygon.c

```diff
@@ -7,6 +7,24 @@
 #include <stddef.h>
 #include <math.h>
 
+static int
+set_polygon_center_coords(pgPolygonBase *polygon)
```

Comment:
i would prefer if this was named `_set_polygon_center_coords` to indicate that is private

Reply:
Agreed